### PR TITLE
Autoscale Table Synces Current Scaleset Settings

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
Currently, AutoScale table entries are not synced with the current state of the VMSS for a OneFuzz instance. This includes config updates and deletes.

## PR Checklist
* [ ] Applies to work item: #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

File: src/ci/api-service-toolcheck.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ set -ex
 
 cd src/api-service
 
-pip install -r requirements-dev.txt
+# pip install -r requirements-dev.txt
```
Comment (mgreisen) [inline src/ci/api-service-toolcheck.sh:10]: Commented out?

Comment (nharper285) [reply src/ci/api-service-toolcheck.sh:10]: Fixed

File: src/api-service/__app__/onefuzzlib/workers/scalesets.py [hunk 2]
```diff
@@ -936,6 +946,75 @@ def set_state(self, state: ScalesetState) -> None:
...
+        scale_out_amount = 1
+        scale_out_cooldown = 10
+        scale_in_amount = 1
+        scale_in_cooldown = 15
+
+        for rule in auto_scale_profile.rules:
+            scale_action = rule.scale_action
+            if scale_action.direction == "Increase":
+                scale_out_amount = scale_action.value
+                logging.info("Number of seconds: %d" % scale_out_cooldown)
```
Comment (tevoinea) [inline src/api-service/__app__/onefuzzlib/workers/scalesets.py:985]: The cooldowns (at least as they are defined on line 977/979) are in minutes. It's possible the response from the azure API is in seconds though.

**Review by tevoinea — APPROVED** [review]
Don't forget to port to C# or mark with `refactor-diverge`

File: src/api-service/__app__/onefuzzlib/workers/scalesets.py (hunk 2 above)
Comment (nharper285) [reply src/api-service/__app__/onefuzzlib/workers/scalesets.py:985]: The response is in seconds.

**Pull request merged**